Website/env-cmdrc

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
Again, this file exposes our API tokens?! Please remove.

Note: Please ask if you don't know how okay?